app/src/components/Register.tsx

```diff
@@ -78,22 +80,30 @@ class Register extends React.Component<RegisterProps, RegisterState> {
     const currentAccount = drizzleState.accounts[0];
     const artist = drizzleState.accounts[0]; // TODO: Update this to real artist's account
 
-    const metaUri = '';
+    const jsonData = {
```

Comment:
Feel like this may also be able to be done directly as they have the same names (or should do)